framework/looper.py

```diff
@@ -3,7 +3,7 @@
 
 import ROOT 
 ROOT.PyConfig.IgnoreCommandLineOptions = True
-
+from heppy.papas.cpp.physicsoutput import PhysicsOutput as  pdebug #alice would prefer not to put it here but I want to output the event #
```

Comment:
remove all modifications to framework classes. 
implement instead a analyzers/test/PDebugAnalyzer